functions/openhab.bash

```diff
@@ -63,12 +63,7 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   if ! cond_redirect apt-get ${APT_INST_OPTS} install "openhab2=${openhabVersion}"; then echo "FAILED (apt)"; exit 1; fi
   cond_redirect adduser openhab gpio
   cond_redirect systemctl daemon-reload
-  if cond_redirect systemctl enable openhab2.service; then echo "OK"; else echo "FAILED (usr)"; exit 1; fi
-  if [ -n "$UNATTENDED" ]; then
```

Comment:
what was this intended for? why are you sure you can remove?

Reply:
See https://github.com/openhab/openhabian/blob/master/functions/openhab.bash#L67
... enable --now ... replaces all of this (and it works as can be seen obn Travis)

Reply:
my question was about the if "unattedded" is ture - I see a stop for the case in the old code. Not we changed to enable and run for both cases.

Reply:
the old ... stop IMHO was a bug, we want it to run in unattended mode, too (else final check will fail in Travis and on real HW, too)